src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -163,7 +163,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.10.0-SNAPSHOT" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="2.1.2" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS6" Version="3.0.630" />
-    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7" Version="3.0.1272" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7" Version="3.0.1573" />
```

Comment:
I'm trying to make the process for updating language workers more consistent in core tools. Please see discussion here: https://github.com/Azure/azure-functions-core-tools/pull/2957#discussion_r812259934

Reply:
Thank you for bringing this up @ejizba. I agree. In addition to the steps you described in https://github.com/Azure/azure-functions-core-tools/pull/2957#discussion_r812314195, I would add: 
4. Have the language worker owner review and sign off on the PR  

In general, we need a way to keep the packages version consistent between the Functions Host and the Core Tools. This is a common pain point for integration testing as well.

Reply:
So to clarify, the PS7 worker is actually too far ahead as it is. It should be 3.0.1045, to match the host's v3.4.2 tag [here](https://github.com/Azure/azure-functions-host/blob/v3.4.2/src/WebJobs.Script/WebJobs.Script.csproj#L54)

Reply:
Given that @michaelpeng36's fix is for the MSI has been merged, we need to do a new retake for both V3 and V4.

As per my discussion with @VpOfEngineering, this PR should be merged after a new cut of the Core Tools is taken for V3. 

Please note that I also have another PR waiting to be merged for V4, which will be merged after a new cut is taken.

Reply:
If you are doing this for the "new cut", I would prefer you close the PowerShell-specific PR and I will ensure whoever updates the host updates the workers at the same time

Reply:
@amamounelsayed @VpOfEngineering -- We should improve in the current process to ensure that all language workers are upgraded to the same version as the Functions Host when cutting a new release. In the current process, the language worker owners update both the Functions Host and the Core Tools for a new worker release. @ejizba has some good ideas here https://github.com/Azure/azure-functions-core-tools/pull/2957#discussion_r812314195 to improve the process. 

@amamounelsayed -- I will let you decide what the right process should be since your team owns deployments. Let me know if you need anything from my side. Thanks.

Reply:
> If you are doing this for the "new cut", I would prefer you close the PowerShell-specific PR and I will ensure whoever updates the host updates the workers at the same time

No, this change should be merged after the new cut is taken. This language worker upgrade will be part of the next release after a new cut is taken.

Reply:
Please see https://github.com/Azure/azure-functions-core-tools/pull/2971 and https://github.com/Azure/azure-functions-core-tools/pull/2972. I have added a script to enforce the host & workers are always updated at the same time and should always match. This means language owners will never need to individually update their worker in this repo. Assuming y'all approve of my first PR, we can close this one.